.eslintrc.json

```diff
@@ -11,9 +11,16 @@
   "rules": {
     "react/prop-types": "off",
     "func-style": "error",
-    "@typescript-eslint/interface-name-prefix": [
+    "@typescript-eslint/naming-convention": [
```

Comment:
is there a reason we're changing this so it no longer matches jupyterlab's lint rules?

Reply:
`@typescript-eslint/interface-name-prefix` has been deprecated in favor of `@typescript-eslint/naming-convention`

Reply:
this probably shouldn't be a part of this PR though